docs/debugging-python-lambda-functions.md

```diff
@@ -51,7 +54,7 @@ These instructions outline how you can debug a lambda handler locally using the
                 "host": "localhost",
                 "pathMappings": [
                     {
-                        "localRoot": "${workspaceFolder}/hello_world",
+                        "localRoot": "${workspaceFolder}/my-sam-app/hello_world",
```

Comment:
It might be worth mentioning assumptions in an intro section to the page. In this case, specifying that these instructions are based on opening WorkspaceFolder to the parent folder of the folder containing the SAM App/Template.

Reply:
Good point. If the user has just created the SAM app using our toolkit, this will already be the case--but we can't assume this.